src/reconcile/rec_row.c

```diff
@@ -206,7 +206,7 @@ __wt_bulk_insert_row(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk)
     WT_RET(__rec_cell_build_leaf_key(session, r, /* Build key cell */
       cursor->key.data, cursor->key.size, &ovfl_key));
     WT_RET(__wt_rec_cell_build_val(session, r, cursor->value.data, /* Build value cell */
-      cursor->value.size, false, WT_TS_NONE, WT_TXN_NONE, WT_TS_MAX, WT_TXN_MAX, 0));
+      cursor->value.size, WT_TS_NONE, WT_TXN_NONE, WT_TS_MAX, WT_TXN_MAX, 0));
```

Comment:
Why doesn't variable-length column-store require the same kind of `__rec_cell_repack()` work as row-store?

Reply:
I can't find the same logic in variable-length column-store. Looks like we repack the cell all the time if I understand the code correctly.

Reply:
I could totally be wrong, but...
```
    820                 /*
    821                  * The key on the old disk image is unchanged. Use time pairs from the cell.
    822                  *
    823                  * FIXME-PM-1524: Currently, we don't store durable_ts in cell, which is a problem
    824                  * we need to solve for prepared transactions. Revisit this in PM-1524.
    825                  */
    826                 durable_ts = newest_durable_ts;
    827                 start_ts = vpack->start_ts;
    828                 start_txn = vpack->start_txn;
    829                 stop_ts = vpack->stop_ts;
    830                 stop_txn = vpack->stop_txn;

    936             last.start_ts = start_ts;
    937             last.start_txn = start_txn;
    938             last.stop_ts = stop_ts;
    939             last.stop_txn = stop_txn;
    940             last.deleted = deleted;
    941             rle = repeat_count;
 ```
and then we loop, or fall out of the main loop, and in both cases end up with something like this:
```
   1142     /* If we were tracking a record, write it. */
   1143     if (rle != 0)
   1144         WT_ERR(__rec_col_var_helper(session, r, salvage, last.value, durable_ts, last.start_ts,
   1145           last.start_txn, last.stop_ts, last.stop_txn, rle, last.deleted, false));
```
Am I missing something?

Reply:
Oh, never mind, I see it now -- the reset transaction IDs are the ones we want to write, so it's all good.

OK, agreed.